casanovo/denovo/evaluate.py

```diff
@@ -310,3 +315,130 @@ def aa_precision_recall(
     n_aa_correct = sum([score > threshold for score in aa_scores_correct])
     n_aa_predicted = sum([score > threshold for score in aa_scores_all])
     return n_aa_correct / n_aa_predicted, n_aa_correct / n_aa_total
+
+
+def _get_true_labels(mgf_filename: str):
+    """
+    Extract a list of peptide labels from an annotated .mgf and make it compatible with depthcharge AA vocabulary
+
+    Parameters:
+    -----------
+    mgf_filename : str
+        Path to the .mgf file to extract from.
+
+    Returns:
+    --------
+    mgf_df : pandas DF
+        The true peptide labels with column "true_seq"
+    """
+    seqs = []
+    with mgf.MGF(mgf_filename) as f_in:
+        for spectrum_dict in f_in:
+            seqs.append(spectrum_dict["params"]["seq"])
+    mgf_df = pd.DataFrame({"true_seq": seqs})
+    mgf_df["true_seq"] = mgf_df["true_seq"].str.replace(
```

Comment:
If 'C' not represented with its permanent modification, this implementation fails to match those with 'C+57.021' token predicted by Casanovo